HW03/ihar_shkuryn/ihar_shkuryn_hw_03_t_02.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class Homework3
+  PATTERN = %r{(?:[0-9]{1,3}\.){3}[0-9]{1,3} - - \[\d{2}+/\w{3}/\d{4}:\d{2}:\d{2}:\d{2} \+\d{4}\]}
+  def task2(log)
+    arr_lines = find_lines_by_pattern log
+    return [] unless arr_lines.size.positive?
+
+    format_text arr_lines
+  end
+
+  def format_text(arr_lines)
```

Comment:
I would write a regular expression for address, ip and datetime and use them to select data from selected rows
up to u